.github/workflows/speakers-corner-issue-validation.yml

```diff
@@ -0,0 +1,40 @@
+name: Speakers-Corner-Issue-Validation
+
+# Manually for now, but will trigger on issue opening
+#on:
+#  workflow-dispatch 
+#  issues:
+#    # The python validation script should not run on edited
+#    types: [opened] #, edited] 
+
+jobs:
+  validate-issue:
+    # Name the Job
+    name: Validate the new issue
+    # Set the type of machine to run on
+    runs-on: ubuntu-latest
+
+  #contains(github.event.issue.labels.*.name, 'bug')
+    steps :
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+        
+      - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install PyGithub
+
+      # Run the validation script only if this is a speaker's corner issue and if it was just opened (validatespeakerscornerissue.py)
+      - name: Validate
+        #if: ${{ contains(github.event.issue.title, 'Speaker') && github.event.issue.types == 'opened' }}
```

Comment:
Let's not rely on the title, but certainly let's use the issue label (`application`)

Also why is this commented?

Reply:
Commented for exactly that reason, just wanted have an example bit of conditional running. Will take care of this.

Reply:
Perfect, I had been wondering about that last bit to prevent an infinite loop when run on edited